build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/maxdome_post_startup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sourceMetaData=/opt/ml/metadata/resource-metadata.json
+
+# Extract the required fields from meta data stored in opt/ml/metadata.
+dataZoneDomainId=$(grep -o '"DataZoneDomainId":"[^"]*"' "$sourceMetaData" | cut -d ':' -f 2 | tr -d '"')
+dataZoneUserId=$(grep -o '"DataZoneUserId":"[^"]*"' "$sourceMetaData" | cut -d ':' -f 2 | tr -d '"')
+dataZoneProjectRepositoryName=$(grep -o '"DataZoneProjectRepositoryName":"[^"]*"' "$sourceMetaData" | cut -d ':' -f 2 | tr -d '"')
+
+# Run AWS CLI command to get the username from DataZone User Profile.
+response=$( aws datazone get-user-profile --domain-identifier "$dataZoneDomainId" --user-identifier "$dataZoneUserId" )
```

Comment:
it might be better to explicitly specify the region . I'm guessing that region is inferred from a profile?

Reply:
+1 lets add the region